lib/action_cable_client.rb

```diff
@@ -162,7 +164,7 @@ def check_for_subscribe_confirmation(message)
     return unless  Message::TYPE_CONFIRM_SUBSCRIPTION == message_type
 
     self._subscribed = true
-    _subscribed_callaback&.call
+    _subscribed_callback&.call
```

Comment:
lol, thanks for these type fixes. :-) :embarrassed:

Reply:
No worries! But now I can't get this out of my head, again.

https://www.youtube.com/watch?v=Kgjkth6BRRY

Reply:
lol :+1: